Admin/Infrastructure/Proxmox/3-RemovingNodes.md

```diff
@@ -1,6 +1,5 @@
 # Removing Nodes
-Author(s):
-    Me
+Author(s):  

```

Comment:
Did you mean to make this have you listed as an author?

(Do we know who "me: is?)

Maybe something like the following
```suggestion
**Author(s)**: 
* Chisom
---
```
* This way "Authors: is bolded 
* The Author's name is indented with a bullet point (Could omit the bullet point)
* The `---` inserts a line